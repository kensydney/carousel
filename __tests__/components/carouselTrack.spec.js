import React from 'react';
import {shallow} from 'enzyme';
import CarouselTrack from '../../src/components/carouselTrack';
import renderer from 'react-test-renderer';


const carousel = {"imageurl": "images/bike.png", "title":"A Guy On A Bike", "synopsis":"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.", "link": "www.good.com"};

describe('<CarouseTrack />', () => {
  it('should show hype link', () => {
    const wrapper = shallow(<CarouselTrack carousel = {carousel} index = {1} activeIndex = {1} />);    
    expect(wrapper.find('a').length).toBe(1);
  });
});

describe('CarouseTrack Snapshot',()=>{
  it('capturing Snapshot of CarouselTrack', () => {
    const renderedValue =  renderer.create(<CarouselTrack carousel={carousel} index = {1} activeIndex = {1} />).toJSON();
    expect(renderedValue).toMatchSnapshot();
  });
});