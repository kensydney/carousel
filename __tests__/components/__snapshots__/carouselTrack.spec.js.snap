// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CarouseTrack Snapshot capturing Snapshot of CarouselTrack 1`] = `
<li>
  <div
    className="carousel-track-container"
  >
    <a
      className="carousel-track carousel-track-active"
      onClick={undefined}
    />
  </div>
</li>
`;
